src/frontend/animals/search/SearchAnimalForm.js

```diff
@@ -11,29 +11,34 @@ class SearchAnimalForm extends Component {
     }
   }
 
+  componentDidMount() {
+    if (!this.props.animals) this.props.getAllAnimals()
+  }
+
   onInputChange(event) {
-    this.props.searchAnimals()
     this.setState({ inputValue: event.target.value })
   }
 
   render() {
-    const { inputValue } = this.state
     return (
       <div>
-        {inputValue}
+        <div>
+          <input
+            placeholder="Enter a name..."
+          />
+        </div>
         {this.props.animals ? this.props.animals.map(animal => (
-          <div key={animal.name}>{animal.name}</div>
+          <div key={animal.name}>
+            {animal.name} {animal.location} {animal.species_name}
```

Comment:
Would be nice if data that gets out of the db would be camelCase, needs a transform func in repository to map non camelcase to camelcase